known-bugs.txt

```diff
@@ -29,436 +29,429 @@ reasons why we think that fixing them is infeasible. We might make some
-	the tiles of a station, instead of just once.
-
-Trains might not stop at platforms that are currently being changed [FS#5553]:
-	If you add tiles to or remove tiles from a platform while a train is
-	approaching to stop at the same platform, that train can miss the place
-	where it's supposed to stop and pass the station without stopping. This
-	is caused by the fact that the train is considered to already have stopped
-	if it's beyond its assigned stopping location. We can't let the train stop
-	just anywhere in the station because then it would never leave the station
-	if you have the same station in the order list multiple times in a row or
-	if there is only one station in the order list (see FS#5684).
-
-Some houses and industries are not affected by transparency [FS#5817]:
-	Some of the default houses and industries (f.e. the iron ore mine) are
-	not affected by the transparency options. This is because the graphics do
-	not (completely) separate the ground from the building.
-	This is a bug of the original graphics, and unfortunately cannot be
-	fixed with OpenGFX for the sake of maintaining compatibility with the
-	original graphics.
-
-Involuntary cargo exchange with cargodist via neutral station [FS#6114]:
-	When two players serve a neutral station at an industry, a cross-company
-	chain for cargo flow can and will be established which can only be
-	interrupted if one of the players stops competing for the ressources of
-	that industry. There is an easy fix for this: If you are loading at the
-	shared station make the order "no unload" and if you're unloading make
-	it "no load". Cargodist will then figure out that it should not create
-	such a route.
+window resizing with SDL on macOS [#3447]:
+        Video handling with (lib)SDL under macOS is known to fail on some
+        versions of macOS with some hardware configurations. Some of
+        the problems happen only under some circumstances whereas others
+        are always present.
+        We suggest that the SDL video/sound backend is not used for OpenTTD
+        in combinations with macOS.
+
+Train crashes entering same junction from block and path signals [#3928]:
+        When a train has reserved a path from a path signal to a two way
+        block signal and the reservation passes a path signal through the
+        back another train can enter the reserved path (only) via that
+        same two way block signal.
+        The reason for this has to do with optimisation; to fix this issue
+        the signal update has to pass all path signals until it finds either
+        a train or a backwards facing signal. This is a very expensive task.
+        The (signal) setups that allow these crashes can furthermore be
+        considered incorrectly signalled; one extra safe waiting point for
+        the train entering from path signal just after the backwards facing
+        signal (from the path signal train) resolves the issue.
+
+Crashes when playing music [#3941]:
```

Comment:
I haven't been able to reproduce crashes with OpenMSX in either of the new music drivers for Windows.